proto/src/determined/api/v1/user.proto

```diff
@@ -57,12 +57,42 @@ message GetUserResponse {
   determined.user.v1.User user = 1;
 }
 
+// Get the requested user by username.
+message GetUserByUsernameRequest {
+  // The string of the username.
+  string username = 1;
+}
+// Response to GetUserByUsernameRequest.
+message GetUserByUsernameResponse {
+  option (grpc.gateway.protoc_gen_swagger.options.openapiv2_schema) = {
+    json_schema: { required: [ "user" ] }
+  };
+  // The requested user.
+  determined.user.v1.User user = 1;
+}
+
+// Get the requested current user.
+message GetMeRequest {
+  // Cannot have an empty message.
```

Comment:
empty request here seems to work: https://github.com/determined-ai/determined/blob/master/proto/src/determined/api/v1/auth.proto#L33